### DISCUSSION TITLE
Site layout question

### DISCUSSION THREAD

**Discussion by Eoldev-Terry** [bassamanator/everything-sovol-sv06#5] [General]
Hi Bassamanator!  I hope that I can contribute.  Do you plan on having everything in the one README.md file, or are you planning to create other .md files for different sections of the printer?  So just asking how you want things to look before I attempt at creating some content.

**Comment by bassamanator** [comment]
Hey!😀 

I think eventually the different major parts/areas of the printer will have their own folders. However, just to keep things simple at this early stage, let's try to keep everything in the main README.md.

Our goal is clarity, ease of access to information, and accuracy of course. 

This is a community project, though, so I'm very open to suggestions. And I can't do this by myself, so thanks for contributing!

**Reply by Eoldev-Terry** [reply]
Hi again.  I don't seem to have the priveleges to create a new branch to add content.  I also don't have push permissions to add images to the images folder.  Maybe you could instruct me on what I'm doing wrong?  I have never contributed to a repository, but I went through Github's tutorial.

**Reply by Eoldev-Terry** [reply]
I found an article on how to do this; I'm going to give it a try.  You should eventually see a pull request if I make it all the way to the end.  Haha.
